[ADD] marketing automation: added 'getting started' section and docs … (#2552)
odoo/documentation: content/applications/marketing/marketing_automation/getting_started/testing_running.rst

Testing & Running
=================

Odoo gives users the ability to test marketing campaigns (and mailings) before officially running
them. 

To start a test, click the 'Launch a Test' button at the top of the template form.

.. image:: testing_running/launch-test.png
   :align: center
   :alt: Launch a test button in Odoo Marketing Automation.

When clicked, a pop-up appears, in which you can choose a specific record to run the test on. Or,
you can create a brand new record to test.

.. image:: testing_running/launch-test-popup.png
   :align: center
   :alt: Launch test popup in Odoo Marketing Automation.

Once you've selected the record you want to test, you'll click 'Continue,' and Odoo will redirect
you to the campaign test page.

.. image:: testing_running/test-screen.png
   :align: center
   :alt: Test screen in Odoo Marketing Automation.

> Here, you'll see the name of the Record you're testing, along with the precise time this workflow
> was started. Beneath that, you'll find your first activity (or activities) in the workflow.

Reviewer: ```suggestion
Here, the name of the :guilabel:`Record` being tested is visible, along with the precise time this
test workflow was started. Beneath that is the first activity (or activities) in the workflow.
```

> To start a test, click the 'Run' icon beside the first activity in the workflow. When clicked, the
> page will reload, and Odoo will show you the various results (and analytics) connected to that
> specific activity.

Reviewer: ```suggestion
To start a test, click the :guilabel:`Run` icon beside the first activity in the workflow. When
clicked, the page will reload, and Odoo will show the various results (and analytics) connected to
that specific activity.
```

Testing & running a campaign
============================

Odoo gives users the ability to test marketing campaigns (and mailings) before officially running
them. 

First, open the :guilabel:`Marketing Automation` application and click on the campaign. Make sure
the campaign already has activities configured on it. To start a test, click the 

> :guilabel:`Launch a Test` button at the top of the template form.

Reviewer 2: Follow up to option 1 @ksc-odoo -- 
Another doc link that might be more appropriate here actually is to point towards `workflow_activities`. If you agree to implementing option 1, feel free to pick whichever preferred doc link that you think is more useful and in context while making sure that the surrounding copy around the link relates to whatever doc you pick 😉

   :align: center
   :alt: Launch a test button in Odoo Marketing Automation.

When clicked, a pop-up appears, in which the user can choose a specific record to run the test on,
or the user can create a brand new record to test.

Once the record is selected, click :guilabel:`Continue`, and Odoo will redirect to the campaign
test page.

.. image:: testing_running/test-screen.png
   :align: center
   :alt: Test screen in Odoo Marketing Automation.

Here, the name of the :guilabel:`Record` being tested is visible, along with the precise time this
test workflow was started. Beneath that is the first activity (or activities) in the workflow.

To start a test, click the :guilabel:`Run` icon beside the first activity in the workflow. When
clicked, the page will reload, and Odoo will show the various results (and analytics) connected to
that specific activity.

.. image:: testing_running/workflow-test-progress.png
   :align: center
   :alt: Workflow test progress in Odoo Marketing Automation.

Once all the workflow activities are completed, the test will end and be moved to the
:guilabel:`Completed` stage. To stop a test before all the workflow activities are completed, click

> the :guilabel:`Stop` button.

Author: There's an empty line break at the end on my version, as well.

> Testing and running a campaign
> ==============================

Reviewer 3: ```suggestion
==============================
Testing and running a campaign
==============================
```

Odoo gives users the ability to test marketing campaigns (and mailings) before officially running
them.

First, open the :guilabel:`Marketing Automation` application and click on a campaign. Make sure
the campaign already has activities configured on it (or build a campaign by following the
directions here on :doc:`this documentation <workflow_activities>`). To start a test, click the
:guilabel:`Launch a Test` button at the top of the template form.

.. image:: testing_running/launch-test.png
   :align: center
   :alt: Launch a test button in Odoo Marketing Automation.

When clicked, a pop-up window appears. In the dropdown field choose a specific record to run the
test on, or create a brand new record by clicking the :guilabel:`Search More...` link at the bottom
of the dropdown menu, and then click the :guilabel:`Create` button.

Once the record is selected, click :guilabel:`Continue`, and Odoo will redirect to the campaign
test page.

> .. image:: testing_running/test-screen.png

Reviewer 2: Image is too wide and not in HD.
Replacement [here](https://i.imgur.com/UkpxOVT.png).

   :align: center
   :alt: Test screen in Odoo Marketing Automation.

Here, the name of the :guilabel:`Record` being tested is visible, along with the precise time this
test workflow was started. Beneath that is the first activity (or activities) in the workflow.

To start a test, click the :guilabel:`Run` icon beside the first activity in the workflow. When
clicked, the page will reload, and Odoo will show the various results (and analytics) connected to
that specific activity.

> .. image:: testing_running/workflow-test-progress.png

Reviewer 2: Too wide.
Replacement [here](https://i.imgur.com/lWiRmER.png).

Testing & running a campaign
============================

Odoo gives users the ability to test marketing campaigns (and mailings) before officially running
them.

First, open the :guilabel:`Marketing Automation` application and click on the campaign. Make sure

> the campaign already has activities configured on it. To start a test, click the :guilabel:`Launch a
> Test` button at the top of the template form.

Reviewer 2: break at or before 100th character:
```suggestion
the campaign already has activities configured on it. To start a test, click the :guilabel:`Launch
a Test` button at the top of the template form.
```

> .. image:: testing_running/launch-test.png

Reviewer 2: This image isn't helpful unless it shows other UI elements around it, otherwise we're saying "randomly look for THIS button". Would retake with other UI elements surrounding it and then highlight the button with red box.

   :align: center
   :alt: Launch a test button in Odoo Marketing Automation.

When clicked, a pop-up appears, in which the user can choose a specific record to run the test on,
or the user can create a brand new record to test.

> .. image:: testing_running/launch-test-popup.png

Reviewer 2: Not sure this screenshot is needed?

Page as a whole is more image-heavy than text...
Wdyt @mivu-odoo ?

Author: I'll remove it right away. Thanks for pointing that out.
